README.md

```diff
@@ -1 +1,44 @@
-# EndToEndReliability
\ No newline at end of file
+# EndToEndReliability
+
+
+## Iteration 1 - The problem
+
+
+### Data going from the client to the server can get lost
+
+Getting data from the browser to the web server is exposed to the fallacies of distributed computing. Lots has been written on that topic but in short:
+
+> things can go wrong, horribly wrong
+
+The request might never reach the server, with an HTTP timeout as a result. It might reach the server but something bad might happen before your data is safely stored, with an HTTP 500 Internal Server Error as a result. It doesn't end there. Things might go wrong after data is stored, and the same 500 will greet the client. Are we done? Nope. Something might go wrong on the way back to the client, and HTTP timeout again. As you can see, lots of things can happen. From the client's perspective we can summarize it as:
```

Comment:
Just an FYI and I'm not sure if we want to mention it. We could tie this also into async and cancelation. Cooperative cancelation in .net means that we might cancel an http client request (because it took too long and we want to retry) but the call is already done to the server. For example we could show a concrete code piece with polly and httpclient factory that demonstrates it

Reply:
We're about to start iteration 2 which covers client side retries so lets see how this fits in there? (will ping for a review on that PR when ready)